app/models/countryside_stewardship_grant.rb

```diff
@@ -6,12 +6,18 @@ class CountrysideStewardshipGrant < Document
     funding_amount
   ).freeze
 
+  COUNTRYSIDE_TAXON_ID = '9129d716-365b-44ba-9856-383423fe1e41'.freeze
```

Comment:
@koetsier This one should be moved to `Document` with it's friends.

Reply:
fixed